src/main/scala/com/github/xiaodongw/swagger/finatra/WebjarsController.scala

```diff
@@ -17,17 +17,18 @@ object WebjarsController {
   private val DEFAULT_EXPIRE_TIME_MS: Long = 86400000L // 1 day
 }
 
-class WebjarsController @Inject() (resolver: FileResolver) extends Controller {
+class WebjarsController (root: String = "/webjars", version: String = "2.2.5") extends Controller {
   import WebjarsController._
 
-  private val root: String = "/webjars"
+  @Inject
+  var resolver: FileResolver = null
+
   private val disableCache: Boolean = false
 
   get(s"${root}/:*") { request: Request =>
     val resourcePath = request.getParam("*")
 
-    val webjarsResourceURI: String = "/META-INF/resources/webjars/" + resourcePath
-    //logger.log(Level.FINE, "Webjars resource requested: {0}", webjarsResourceURI)
+    val webjarsResourceURI: String = s"/META-INF/resources/webjars/swagger-ui/$version/$resourcePath"
```

Comment:
Before the WebjarsController is general to support all web jars, now it ties to swagger-ui.